app/src/main/java/com/ahmettekin/WeatherApp/adapter/RecyclerViewAdapter.java

```diff
@@ -101,7 +104,7 @@ private void configureUI(WeatherItem weatherItem) {
             tvWeatherTemp.setText(MessageFormat.format("Sıcaklık : {0}{1}", weatherItem.getMain().getTemp(), DEGREE_SYMBOL));
             tvSky.setText(MessageFormat.format("Gökyüzü : {0}", UpperCaseWords(weatherItem.getWeather()[0].getDescription())));
```

Comment:
1. getWeather -> getWeathers olarak değiştirelim
2. getWeather[0] elemanı 107.satırda da kullanılmış.Bunu configureUI'da en başta değişkene alıp oradan çağıralım.
3. ve değişkene aldığımız yerde index of bounds hatası alınmasına karşı count > 0 kontrolü yapalım(crash olmasını önlemek için)